### PR TITLE
[Merged by Bors] - feat(Topology.Algebra.Valuation): add integer_isOpen

### PR DESCRIPTION
**Pull request by mariainesdff**
The unit ball in a valued ring is open.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Valuation.lean [hunk 1]
```diff
@@ -168,4 +168,23 @@ theorem cauchy_iff {F : Filter R} : Cauchy F ↔
...
+theorem integer_isOpen : IsOpen (_i.v.integer : Set R) := by
+  rw [isOpen_iff_mem_nhds]
+  intro x hx
+  rw [SetLike.mem_coe, mem_integer_iff] at hx
+  rw [mem_nhds]
+  use (1 : Units Γ₀)
+  intro y hy
+  rw [Units.val_one, mem_setOf_eq] at hy
+  rw [SetLike.mem_coe, mem_integer_iff, ← sub_add_cancel y x]
+  exact le_trans (map_add _ _ _) (max_le (le_of_lt hy) hx)
```
Comment (mo271) [inline Mathlib/Topology/Algebra/Valuation.lean:183]:
```suggestion
  rw [mem_nhds]
  exact ⟨1,
      fun y hy => (sub_add_cancel y x).symm ▸ le_trans (map_add _ _ _) (max_le (le_of_lt hy) hx)⟩
```

**Review by mo271 — APPROVED** [review]
golfed a bit, otherwise LGTM

**Review by jcommelin — COMMENTED** [review]
bors d+

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**